samples/star/src/main/kotlin/com/slack/circuit/star/petdetail/PetDetail.kt

```diff
@@ -200,46 +201,52 @@ private fun UnknownAnimal(paddingValues: PaddingValues) {
 private fun ShowAnimal(
   state: PetDetailScreen.State.Success,
   padding: PaddingValues,
-) =
-  when (LocalConfiguration.current.orientation == Configuration.ORIENTATION_LANDSCAPE) {
-    true -> ShowAnimalLandscape(state, padding)
-    false -> ShowAnimalPortrait(state, padding)
+) {
+  val carouselContent = remember {
+    movableContentOf {
```

Comment:
Just curious: what is `movableContentOf` being used for here? Is it because you have created a `@Composable` lambda here that is used by both portrait and landscape logic?

Reply:
Because keys are positional in compose, we use this to handle the fact that this content moves positions in the slot table depending on the orientation. Basically telling compose "hey this is moving, not a different composition"